ios/FluentUI/Button/Button.swift

```diff
@@ -300,15 +320,23 @@ open class Button: UIButton {
     }
 
     private func normalTitleAndImageColor(for window: UIWindow) -> UIColor {
-        return style == .primaryFilled ? Colors.Button.titleWithFilledBackground : Colors.primary(for: window)
+        if style.isFilledStyle {
+            return Colors.Button.titleWithFilledBackground
```

Comment:
I may not understand the interaction of the various states/how colors are handle, but does this mean that any filled style (including a danger filled style) will return the titleWithFilledBackground color? Should you be checking for isDangerStyle before isFilled style?

Reply:
Correct. The text will have the same color for both primaryFilled and dangerFilled. No need to check because the color assigned is the same:
![Screen Shot 2021-11-29 at 2 40 42 PM](https://user-images.githubusercontent.com/68076145/143954093-6721cb2f-bbb9-46ee-8453-0a1e02181d28.png)
![Screen Shot 2021-11-29 at 2 40 52 PM](https://user-images.githubusercontent.com/68076145/143954103-6d278269-b9a9-456e-9a08-52cacee5ce27.png)

The other check for isDangerStyle covers the outlined versions:

![Screen Shot 2021-11-29 at 2 43 10 PM](https://user-images.githubusercontent.com/68076145/143954338-60796e54-c3c0-4dc3-9b35-46df69ed9983.png)
![Screen Shot 2021-11-29 at 2 43 19 PM](https://user-images.githubusercontent.com/68076145/143954352-944b4b9e-2b2a-4d68-a904-254c58485738.png)